Kentico.Kontent.Management.Tests/Unit/Modules/UrlBuilder/AssetTests.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Kentico.Kontent.Management.Models.Shared;
+using System;
+using Xunit;
+
+namespace Kentico.Kontent.Management.Tests.Unit.Modules.UrlBuilder
+{
+    public partial class EndpointUrlBuilderTests
```

Comment:
I think we should separate tested method in test name with '_'. It might improve readability.
Shouldn't test name include externalId if tests use externalId?

Reply:
Will be handled in a separate issue.